package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/UrpShaderImporter.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2023 Unity Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+
+namespace Unity.Formats.USD
+{
+    public class UrpShaderImporter : ShaderImporterBase
+    {
+        public UrpShaderImporter(Material material) : base(material)
+        {
+        }
+
+        // Based on HdrpShaderImporter.cs, reordered to the same order as URP's Lit.shader
+        public override void ImportFromUsd()
+        {
+            Material mat = Material;
+
+            mat.SetFloat("_WorkflowMode", IsSpecularWorkflow ? 0.0f : 1.0f);
+
+            // BaseMap
+            if (DiffuseMap)
+            {
+                mat.SetTexture("_BaseMap", DiffuseMap);
+                mat.SetColor("_BaseColor", Color.white);
+            }
+            else
+            {
+                mat.SetColor("_BaseColor", Diffuse.GetValueOrDefault(mat.color));
+            }
+
+            // AlphaCutoff - ignored
+
+            // Smoothness
+            // TODO: We could alternatively bake the roughness into the albedo alpha channel. However this should be
+            // optional because we shouldn't silently overwrite the albedo alpha channel.
+            Texture2D metallicGloss = null;
+            if (RoughnessMap && !IsSpecularWorkflow && MetallicMap)
+            {
+                // If we have and are using a metallic map, combine the roughness into the metallic map alpha channel.
+                // CombineRoughness also flips rough to smooth.
+                metallicGloss = MaterialImporter.CombineRoughness(MetallicMap, RoughnessMap, "metallicGloss");
+                mat.EnableKeyword("_METALLICSPECGLOSSMAP");
+                mat.SetFloat("_SmoothnessTextureChannel", 0.0f);
+            }
+            else
+            {
+                // TODO: In URP we combine a smoothness map with a constant smoothness value, so we should actually
```

Comment:
Do we already have a ticket for this?

Reply:
added https://jira.unity3d.com/browse/USDU-474